test/tuwien/auto/calimero/dptxlator/DptXlator16BitSetTest.java

```diff
@@ -77,6 +78,29 @@ void init() throws Exception {
 		t = new DptXlator16BitSet(DptXlator16BitSet.DptRhccStatus);
 	}
 
+	@Test
+	void testSet() throws KNXFormatException {
+		t = new DptXlator16BitSet(DptXlator16BitSet.DptChannelActivation16);
+		t.setValue("1 1 0 0 0 0 1 0 0 0 0 0 1 1 0 1");
+		assertEquals("channel16, channel15, channel10, channel4, channel3, channel1", t.getValue());
```

Comment:
it's not relevant for this PR, but I now notice that the pretty-printing puts no space between a string and number

Reply:
as you can use ```setValue()``` with this pretty printed string, it might be a breaking change.....

Reply:
one can also directly use the enum constants name, so there is no visible difference.